The Asset Manager is now visible (#330)
akeneo/pim-api-docs: content/concepts/catalog-structure.md

# Catalog structure

> In the PIM, products are structured thanks to the Catalog structure enities.

Author: ```suggestion
In the PIM, products are structured thanks to the Catalog structure entities.
```

> Each section below contains an explanation of the concept behind these resources, what is their usage in the PIM and their JSON format in order to interact with the API.

Reviewer: ```suggestion
Each section below contains an explanation of the concept behind these resources. You will find out more about their usage in the PIM and their JSON format in order for them to interact with the API.
```
